chapters/external-repository-identifiers.md

```diff
@@ -238,15 +238,33 @@ Examples:
 
 External Reference Site: [Software Heritage persistent IDentifiers](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
-### F.4.1 gitbom <a name="gitbom"></a>
+### F.4.2 gitoid <a name="gitoid"></a>
 
-Locator Format: gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>
-	
-Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+Locator Format:
+
+```
+"gitoid:"<git object type>":"<hash algorithm>":"<hash value>
```

Comment:
Looks like the scheme syntax was changed with the reformat from inline code to code on a new line. Updating to ensure consistency w/ the scheme in the IANA link
```suggestion
gitoid":"<git object type>":"<hash algorithm>":"<hash value>
```

Reply:
No, I made a comment before. The literal string is `"gitoid:"`, not just the colon.

Reply:
I tend to concur here, the literal string is "gitoid:".

Reply:
Got it. Looks like the gitoid scheme in the IANA URI filing is what should change. Good catch, @zvr